azurerm/resource_arm_signalr_service.go

```diff
@@ -72,6 +72,48 @@ func resourceArmSignalRService() *schema.Resource {
 				},
 			},
 
+			"features": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
running the tests it appears this is also defaulted by Azure:

```
------- Stdout: -------
=== RUN   TestAccAzureRMSignalRService_capacityUpdate
=== PAUSE TestAccAzureRMSignalRService_capacityUpdate
=== CONT  TestAccAzureRMSignalRService_capacityUpdate
--- FAIL: TestAccAzureRMSignalRService_capacityUpdate (247.31s)
    testing.go:569: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: azurerm_signalr_service.test
          cors.#:                      "1" => "0"
          cors.0.allowed_origins.#:    "1" => ""
          cors.0.allowed_origins.0:    "*" => ""
          features.#:                  "1" => "0"
          features.0.flag:             "ServiceMode" => ""
          features.0.value:            "Default" => ""
          hostname:                    "acctestsignalr-191025150234218152.service.signalr.net" => "acctestsignalr-191025150234218152.service.signalr.net"
          id:                          "/subscriptions/1a6092a6-137e-4025-9a7c-ef77f76f2c02/resourceGroups/acctestRG-191025150234218152/providers/Microsoft.SignalRService/SignalR/acctestSignalR-191025150234218152" => "/subscriptions/1a6092a6-137e-4025-9a7c-ef77f76f2c02/resourceGroups/acctestRG-191025150234218152/providers/Microsoft.SignalRService/SignalR/acctestSignalR-191025150234218152"
          ip_address:                  "13.93.42.15" => "13.93.42.15"
          location:                    "westeurope" => "westeurope"
          name:                        "acctestSignalR-191025150234218152" => "acctestSignalR-191025150234218152"
          primary_access_key:          "oU+NldB5H13VX8+ZMsB2hPZskelTQYaAdRFGkikhI78=" => "oU+NldB5H13VX8+ZMsB2hPZskelTQYaAdRFGkikhI78="
          primary_connection_string:   "Endpoint=https://acctestsignalr-191025150234218152.service.signalr.net;AccessKey=oU+NldB5H13VX8+ZMsB2hPZskelTQYaAdRFGkikhI78=;Version=1.0;" => "Endpoint=https://acctestsignalr-191025150234218152.service.signalr.net;AccessKey=oU+NldB5H13VX8+ZMsB2hPZskelTQYaAdRFGkikhI78=;Version=1.0;"
          public_port:                 "443" => "443"
          resource_group_name:         "acctestRG-191025150234218152" => "acctestRG-191025150234218152"
          secondary_access_key:        "wVi4jHPQ7D8ana5Mh5Rd8Bp166/xsKYRHkgOvU1Zueo=" => "wVi4jHPQ7D8ana5Mh5Rd8Bp166/xsKYRHkgOvU1Zueo="
          secondary_connection_string: "Endpoint=https://acctestsignalr-191025150234218152.service.signalr.net;AccessKey=wVi4jHPQ7D8ana5Mh5Rd8Bp166/xsKYRHkgOvU1Zueo=;Version=1.0;" => "Endpoint=https://acctestsignalr-191025150234218152.service.signalr.net;AccessKey=wVi4jHPQ7D8ana5Mh5Rd8Bp166/xsKYRHkgOvU1Zueo=;Version=1.0;"
          server_port:                 "443" => "443"
          sku.#:                       "1" => "1"
          sku.0.capacity:              "1" => "1"
          sku.0.name:                  "Standard_S1" => "Standard_S1"
```

.. as such we'll need to make this:

```suggestion
				Optional: true,
				Computed: true,
```